website/static/examples/next/drawer-actions.js

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { View, Button, Text } from 'react-native';
+import { NavigationNativeContainer } from '@react-navigation/native';
+import {
+  createDrawerNavigator,
+  DrawerContentScrollView,
+  DrawerItemList,
+  DrawerItem,
+} from '@react-navigation/drawer';
+import { DrawerActions } from '@react-navigation/routers';
+
+const navRef = React.createRef(null);
+
+function HomeScreen({ navigation }) {
+  const jumpToAction = DrawerActions.jumpTo('Profile', { user: 'Satya' });
+
+  return (
+    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+      <Text>Home!</Text>
+      <Button
+        title="Open Drawer"
+        onPress={() => navigation.dispatch(DrawerActions.openDrawer())}
+      />
+      <Button
+        title="Toggle Drawer"
+        onPress={() => navigation.dispatch(DrawerActions.toggleDrawer())}
+      />
+      <Button
+        title="Jump to Profile"
+        onPress={() => navigation.dispatch(jumpToAction)}
+      />
+    </View>
+  );
+}
+
+function ProfileScreen({ route }) {
+  return (
+    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+      <Text>Profile!</Text>
+      <Text>{route?.params?.user ? route.params.user : 'Noone'}'s profile</Text>
+    </View>
+  );
+}
+
+function CustomDrawerContent(props) {
+  return (
+    <DrawerContentScrollView {...props}>
+      <DrawerItemList {...props} />
+      <DrawerItem
+        label="Close drawer"
+        onPress={() => navRef.current.dispatch(DrawerActions.closeDrawer())}
```

Comment:
Maybe `useNavigation()` and then just `openDrawer()`

Reply:
You also get `navigation` in props

Reply:
I will use navigation from props then :)